generators/generator-base.js

```diff
@@ -46,8 +46,8 @@ Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enab
             file: fullPath,
             needle: 'jhipster-needle-add-element-to-menu',
             splicable: [
-                '<li ui-sref-active="active" >\n' +
-                '                    <a ui-sref="' + routerName + '" ng-click="vm.collapseNavbar()">\n' +
+                '<li uiSrefActive="active" >\n' +
```

Comment:
I reverted those lines in the generator. Tests pass now.
Is there a way to run the e2e tests locally (without travis)?